metadata-ingestion/tests/integration/mysql/test_mysql.py

```diff
@@ -50,7 +50,7 @@ def test_mysql_ingest_no_db(
     # Run the metadata ingestion pipeline.
     config_file = (test_resources_dir / "mysql_to_file_no_db.yml").resolve()
     run_datahub_cmd(
-        ["ingest", "--strict-warnings", "-c", f"{config_file}"], tmp_path=tmp_path
+        ["ingest", "--no-strict-warnings", "-c", f"{config_file}"], tmp_path=tmp_path
```

Comment:
def confused why this is necessary when it wasn't there initially

Reply:
trying to investigate